FlowCrypt/Models/SignIn Models/SignInType.swift

```diff
@@ -9,19 +9,19 @@
 import UIKit
 
 enum SignInType: String {
-    case gmail, outlook
+    case gmail, other
 
     var title: String {
         switch self {
         case .gmail: return "sign_in_gmail".localized
-        case .outlook: return "sign_in_outlook".localized
+        case .other: return "sign_in_other".localized
```

Comment:
We still plan to have a dedicated Outlook sign in, which will use oAuth just like Gmail.

Please leave the Outlook button there and add a third button for other.